software/controller/lib/core/sensors.h

```diff
@@ -65,7 +65,7 @@ class Sensors {
   SensorReadings get_readings() const;
 
  private:
-  /// \TODO: get this either from IDC constants header or something like that
+  /// \TODO: get this either from ADC constants header or something like that
```

Comment:
I think I meant to say "ICD", so it was a typo but in different direction. But maybe your meaning is more correct?